packages/babel-core/test/api.js

```diff
@@ -667,19 +667,19 @@ describe("api", function () {
   });
 
   it("code option false", function () {
-    return transformAsync("foo('bar');", { code: false }).then(function (
-      result,
-    ) {
-      expect(result.code).toBeFalsy();
-    });
+    return transformAsync("foo('bar');", { code: false }).then(
+      function (result) {
+        expect(result.code).toBeFalsy();
+      },
+    );
```

Comment:
@sosukesuzuki I don't remember we made changes to affect this, maybe you do?

Reply:
I don't remember...

Reply:
This feels like an improvement to me tbh.